images/cfw/usr/etc/system/webhmi/little_webhmi.py

```diff
@@ -0,0 +1,112 @@
+# aiohttp variant
+import aiohttp
+from aiohttp import web
+import asyncio
+import subprocess
+import json
+import threading
+import aiohttp_jinja2
+import jinja2
+import os
+
+# Print the current working directory for debugging
+print("Current working directory:", os.getcwd())
```

Comment:
NB: X1Plus packages live in the `/opt/x1plus/lib/python/` directory.  So this probably should be `x1plus.services.x1plusd.webhmi` or something.